.gitignore

```diff
@@ -7,3 +7,5 @@ gen
 project.properties
 gradle.properties
 local.properties
+*iml
+/build.gradle
```

Comment:
Whenever I synced to the gradle file a root build.gradle was created and the various .iml files. I thought this would affect other people too, so added them to the .gitignore. I probably shouldn't include this in a PR as an unrelated change though. Sorry, happy to take it out of this branch if you would like me to.